python/ray/util/client/worker.py

```diff
@@ -313,31 +314,39 @@ def wait(self,
 
         return (client_ready_object_ids, client_remaining_object_ids)
 
-    def call_remote(self, instance, *args, **kwargs) -> List[bytes]:
+    def call_remote(self, instance, *args, **kwargs) -> List[Future]:
```

Comment:
Can you make sure that `call_remote` in api.py also has the updated type? I think it might also be good to change `call_remote` in api.py to `_call_remote` to make it clear that it is not for public usage.

Reply:
A TODO is totally fine! I think moving all of those to internal would be good in a follow up :)

Reply:
```suggestion
    def call_remote(self, instance, *args, **kwargs) -> List[Future]:
    # TODO(mwtian) Refactor `call_remote`, `call_release` & `call_retain`
    # to have internal prefix
```